setup.py

```diff
@@ -1,4 +1,19 @@
 from setuptools import setup
+from pathlib import Path
+import re, ast
```

Comment:
please split over multiple lines, and sort the imports

Reply:
changes have been made, thanks for reviewing!